mne/preprocessing/ica.py

```diff
@@ -254,7 +254,8 @@ def __repr__(self):
               hasattr(self, 'n_components_') else
               'no dimension reduction')
         if self.info is not None:
-            ch_fit = ['"%s"' % c for c in ['mag', 'grad', 'eeg'] if c in self]
+            ch_fit = ['"%s"' % c for c in ['mag', 'grad', 'eeg', 'seeg']
```

Comment:
instead of having `['mag', 'grad', 'eeg', 'seeg']` here you should:

1. add `_DATA_TYPES_SPLIT = ('meg', 'eeg', 'seeg')` as a constant in whatever file `_pick_data_channels` lives in
2. import `_DATA_TYPES_SPLIT` in this file
3. replace both instances of `['mag', 'grad', 'eeg', 'seeg']` in this file with `_DATA_TYPES_SPLIT`

It moves us toward having one place we specify what "data channels" mean